style.css

```diff
@@ -69,3 +123,295 @@ h1 span {
+  width: 48px;
+  height: 4px;
+  background: #36b37e;
+  border-radius: 24px;
+  display: flex;
+  align-items: center;
+  margin-left: 160px;
+  margin-right: 135.5px;
+  margin-top: 5px;
+}
+
+.container2 h2 {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  font-size: 40px;
+  font-weight: 700;
+  margin-left: 78px;
+  color: #172b4d;
+}
+
+.first-img {
+  background: url(images/Snapshoot\ Portfolio.png) no-repeat;
+  width: 695vh;
+  height: 220px;
+  margin-left: 10%;
+  margin-top: 50px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+}
+
+.container2 h3 {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  font-size: 24px;
+  color: #3a4a42;
+  margin-left: 80px;
+  padding-top: 10px;
+  margin-bottom: 10px;
+  line-height: 32px;
+}
+
+.container2 ul {
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.container2 a button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 15px;
+  margin-left: 115px;
+  margin-bottom: 48px;
+  margin-top: 48px;
+  color: #fff;
+  border: #fff;
+  gap: 10px;
+  width: 122px;
+  height: 48px;
+  bottom: 8px;
+  background: #36b37f;
+  border-radius: 8px;
+}
+
+.container2 a button:hover {
+  color: #1e488f;
+}
+
+.container2 .second-img {
+  background: url(images/snapa.png) no-repeat;
+  width: 295px;
+  height: 220px;
+  margin-left: 10%;
+  margin-top: 0;
+}
+
+.container2 .third-img {
+  background: url(images/Snapb.png) no-repeat;
+  width: 295px;
+  height: 220px;
+  margin-left: 10%;
+  margin-top: 0;
+}
+
+.container2 .fourth-img {
+  background: url(images/Snapc.png) no-repeat;
+  width: 295px;
+  height: 220px;
+  margin-left: 10%;
+  margin-top: 50px;
+}
+
+.container2 .fifth-img {
+  background: url(images/snapa.png) no-repeat;
+  width: 295px;
+  height: 220px;
+  margin-left: 10%;
+  margin-top: 0;
+}
+
+.container2 .sixth-img {
+  background: url(images/Snapb.png) no-repeat;
+  width: 295px;
+  height: 220px;
+  margin-left: 10%;
+  margin-top: 0;
+}
+
+.container3 h2 {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  font-size: 40px;
+  position: absolute;
+  font-weight: 700;
+  margin-left: 80px;
+  color: #172b4d;
+  margin-top: -65px;
+}
+
+.container3 p {
+  font-size: 20px;
+  display: flex;
+  text-align: center;
+  line-height: 1.5;
+  color: #42526e;
+  padding-top: 15px;
+}
+
+.my-resume {
+  display: flex;
+  align-items: center;
+  color: #fff;
+  border: #fff;
+  gap: 10px;
+  width: 150px;
+  height: 48px;
+  bottom: 8px;
+  padding: 10px 10px;
+  margin-left: 110px;
+  background: #36b37f;
+  letter-spacing: 1px;
+  border-radius: 4px;
+}
+
+.last-img img {
+  width: auto;
+  height: auto;
+  margin-left: 290px;
+}
+
+.container3 a {
+  padding: 20px;
+}
+
+.last-img2 img {
+  display: flex;
+  position: absolute;
+  margin-top: -80px;
+  margin-left: -13px;
```

Comment:
- [ ] The section has the margins (top, bottom, left, right) defined in the design,You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). please kindly align the modal images ,Languages and technologies at the same center lane as the figma design template ,please fix this.
- [ ] Figma design
![Screenshot 2023-06-10 113449](https://github.com/AhmedMh0/Mobile-Portfolio/assets/92458236/b350e816-874c-408a-b34f-2fd023d22846)

- [ ] your design
![Screenshot 2023-06-10 114218](https://github.com/AhmedMh0/Mobile-Portfolio/assets/92458236/1f27c238-3180-49dd-8207-0bc1e2852cc7)